src/js/common/components/RequiredLoginView.jsx

```diff
@@ -141,17 +99,17 @@ class RequiredLoginView extends React.Component {
       view = <LoadingIndicator setFocus message="Loading your information"/>;
     } else {
       if (this.props.authRequired === 1) {
-        if (this.state.accountType >= 1) {
+        if (this.props.userProfile.accountType >= 1) {
```

Comment:
`auhtRequired` is the LOA required to access that app `accountType` is the LOA that that user has. So an example is that a user that has an LOA of 3 can access any app that only requires LOA of 1 because they are greater than 1.